Formula/nmh.rb

```diff
@@ -1,16 +1,10 @@
 class Nmh < Formula
   desc "The new version of the MH mail handler"
   homepage "http://www.nongnu.org/nmh/"
-  url "https://download.savannah.gnu.org/releases/nmh/nmh-1.6.tar.gz"
-  sha256 "29338ae2bc8722fe8a5904b7b601a63943b72b07b6fcda53f3a354edb6a64bc3"
-
-  bottle do
-    sha256 "7c17be97fcf0b6b79c9142887ce966868ed07c856bdd5e0ecdd9a2767a66fc51" => :high_sierra
-    sha256 "62b6f1754f82dcf84674eeddfe2bf7637262324b384e6d90a52bd2a694061da0" => :sierra
-    sha256 "8c508d92233154eb888357f62d84440526fd2acea65566ef5d3325fd0b57caa8" => :el_capitan
-    sha256 "bdd5c9d0e4bd341df41972b4e651761798660eadbac1332409e184e5e8e2b877" => :yosemite
-  end
+  url "https://download.savannah.gnu.org/releases/nmh/nmh-1.7.tar.gz"
+  sha256 "cd05c7ca2cae524ae99f6ba673463a5cdeff62df93e85913aa9277ae8304ce44"
 
+  bottle :disable, "Needs to look for helper programs at compile time"
```

Comment:
We'll need to figure out how to keep this bottled as `bottle :disable` is not permitted except in extreme cases such as cryptopp
```
  # https://cryptopp.com/wiki/Config.h#Options_and_Defines
  bottle :disable, "Library and clients must be built on the same microarchitecture"
```